tests/legacy-cli/e2e_runner.ts

```diff
@@ -175,13 +175,22 @@ Promise.all([findFreePort(), findFreePort()])
       await runSteps(runSetup, allSetups, 'setup');
       await runSteps(runInitializer, allInitializers, 'initializer');
       await runSteps(runTest, testsToRun, 'test');
-
+    } finally {
+      registryProcess.kill();
```

Comment:
This would cause the NPM registry to be terminated as well when debug is enabled.

Reply:
Good point 👍